Header header
float32[] positionX
float32[] positionY
float32[] laserX
float32[] laserY
string[] objects
float32[] objectsX
float32[] objectsY